src/main/scala/battleship/routes/gameActor.scala

```diff
@@ -0,0 +1,84 @@
+package battleship.routes

```

Comment:
The other actors, for as far as they are currently implemented, are in the battleship package, so I'd place this there as well. We could discuss how to organise the classes later, but that's perhaps good to discuss in the group